record-and-playback/core/lib/xml_parser.rb

```diff
@@ -0,0 +1,50 @@
+# BigBlueButton open source conferencing system - http://www.bigbluebutton.org/
```

Comment:
This isn't a good filename, and this isn't the right place to put this file, I think.

This isn't a generic xml parsing library, but really a helper for the bbb-conf script. So it should have a name that reflects that.

The files needed to run bbb-conf all need to be installed as part of the bbb-config package, while if you put it here then it will get installed as part of the bbb-record-core package.

Reply:
Ohh, Ok, I get it, so do you have a suggestion on where I could put that file? And about the name: I am changing it to `xml_helper_conf.rb` if that's ok!

Reply:
This file hasn't yet been moved/removed from the PR.